deploy/docs/Troubleshoot_Collection.md

```diff
@@ -227,6 +227,29 @@ Relevant Fluentd metrics include:
 
 ## Common Issues
 
+### Missing metrics - cannot see cluster in Explore
+
+If your are not seeing metrics coming in to Sumo or/and your cluster is not showing
+up in [Explore][explore] it is most likely due to the fact that Prometheus pod
+isn't running.
```

Comment:
I suggest to use one style, either longer form `is not/are not` or shorter `isn't/aren't`.